stable/eventstore/values.yaml

```diff
@@ -124,6 +124,17 @@ persistence:
   mountPath: /var/lib/eventstore
   annotations: {}
 
+## Export prometheus metrics
+metrics:
+  enabled: false
+  image:
```

Comment:
Could we un-nest this and remove the defaults? I would prefer to make 
the image values explicit and make it consistent with the other image values:
```yaml
metrics:
  image: marcinbudny/eventstore_exporter
  imageTag: 0.7.0
```
You could always add a different registry in the `image` value. The default for no registry is already `docker.io` so no need to specify.

Reply:
I've moved the image values up a level and made their naming consistent with the other images. Not sure if I'm understanding the ask to "remove the defaults" though; do you want the `image` and `imageTag` to be set to the empty string?